fvm/evm/stdlib/contract.cdc

```diff
@@ -9,33 +11,45 @@ contract EVM {
         access(all)
         let bytes: [UInt8; 20]
 
-        /// Constructs a new EVM address from the given byte representation.
+        /// Constructs a new EVM address from the given byte representation
         init(bytes: [UInt8; 20]) {
             self.bytes = bytes
         }
+
+        /// Deposits the given vault into the EVM account with the given address
+        access(all)
+        fun deposit(from: @FlowToken.Vault) {
+            let amount = from.balance
+            destroy from
```

Comment:
I guess this is another thing that makes the total supply field on the FlowToken contract useless.

Reply:
Not sure I understand, can you elaborate?

Reply:
`destroy from` will update the total supply of FlowTokens, but they are not actually being burned, just transformed into another form. Which means `FlowToken.totalSupply` wont be useful anymore, as it looses its meaning.

Reply:
This has been resolved in a follow up PR, #4986, see https://github.com/onflow/flow-go/pull/4986/files#diff-02837edc597afad5bd8f7d1962c129bded25adcc21a9889382c7aaad9a0e5b34L23-R26